src/en/WuxiaDotBlog.lua

```diff
@@ -63,7 +63,8 @@ return {
 			end)
 		end)
 	},
-
+	shrinkURL = shrinkURL,
+        expandURL = function(url) return baseURL .. url end,
```

Comment:
This line uses spaces whereas the rest of the file uses tabs. Additionally, I wouldn't compress this function to a single line.

```suggestion
	expandURL = function(url)
		return baseURL .. url
	end,
```